src/layer/AbstractSentinelHubV1OrV2Layer.ts

```diff
@@ -140,4 +155,53 @@ export class AbstractSentinelHubV1OrV2Layer extends AbstractLayer {
 
     return response.data.map((date: string) => moment.utc(date).toDate());
   }
+
+  public async getStatsAndHistogram(params: GetStatsAndHistogramParams): Promise<StatsAndHistogram> {
+    if (!params.geometry) {
+      throw new Error('Parameter "geometry" needs to be provided');
+    }
+    if (!params.crs) {
+      throw new Error('Parameter "crs" needs to be provided');
+    }
+    if (!params.resolution) {
+      throw new Error('Parameter "resolution" needs to be provided');
+    }
+    if (!params.fromTime || !params.toTime) {
+      throw new Error('Parameter "fromTime" and "toTime" needs to be provided');
```

Comment:
`'Parameters "fromTime" and "toTime" need to be provided'`

Reply:
Changed in `AbstractSentinelHubV1OrV2Layer.ts ` and `AbstractSentinelHubV3Layer.ts `